src/main/java/templates/chat.soy

```diff
@@ -45,9 +45,9 @@
                         <h2 class="mdc-typography--headline5 groups-headline">Groups</h2>
 
                         <div class="selection-controls-container">
-                            {for $group in $groups}
-                                <a class="mdc-list-item" href="/chat" tabindex="0">
-                                    <span class="mdc-list-item__text group-label">{$group}</span>
+                            {for $groupMap in $groups}
+                                <a class="mdc-list-item" href="/chat?groupID={$groupMap['groupID']}" tabindex="0">
```

Comment:
groupID will be changed to groupId in StringConstants, and I think that parameters in the query string are case sensitive. could you change it wherever it's being used?